sdk/servicebus/service-bus/src/receivers/receiver.ts

```diff
@@ -132,15 +134,133 @@ export interface ServiceBusReceiver<ReceivedMessageT> {
    * Use the `createReceiver()` method on the ServiceBusClient to create a new Receiver.
    */
   close(): Promise<void>;
+  /**
+   * Removes the message from Service Bus.
+   *
+   * @throws Error with name `SessionLockLostError` (for messages from a Queue/Subscription with sessions enabled)
+   * if the AMQP link with which the message was received is no longer alive. This can
+   * happen either because the lock on the session expired or the receiver was explicitly closed by
+   * the user or the AMQP link got closed by the library due to network loss or service error.
```

Comment:
```suggestion
   * the user or the AMQP link is closed by the library due to network loss or service error.
```
?

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/11962/commits/3ec99efacab643ad6fa9ddae7c225ebbbb7231b2